### ISSUE TITLE
Request: deploy-service artefact available on maven.org

### ISSUE THREAD

**Issue by kemra102** [pinterest/teletraan#54]
Can we get a deployable artefact for `deploy-service` published to maven.org?

**Comment by sbaogang** [comment]
Absolutely! Thanks for your input. We will work on publish the deploy-service jar into public maven repo sometime next week.

**Comment by mxk1235** [comment]
any update on this?

**Comment by sbaogang** [comment]
We currently only build SNAPSHOT version, looks like the maven central does not allow that: https://maven.apache.org/guides/mini/guide-central-repository-upload.html, we need figure out a good release strategy to publish release versions - let us know if there is doc on how to do that. I assume this is the repo we want to use: http://search.maven.org/

**Comment by mxk1235** [comment]
gotcha.  thanks!

**Comment by sbaogang** [comment]
That said, we will work on the release process. It is just we imagine most people might not run on top of the jar directly. if teletraan is a package people depends on, it would make more sense - but again, we will release it in mavenrepo once we have enough bandwidth and understand how we can release jars not through snapshots.
